.claude-plugin/README.md

```
@@ -0,0 +1,97 @@
+
+Verify the install before enabling the plugin:
+
+```bash
+python -m gradata --version
+```
+
+If `python` is not on your `PATH`, or maps to Python < 3.11, fix that first — the plugin's hooks shell out to `python -m gradata.hooks.*` and will fail silently otherwise.
+
+## Install
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)